src/contrib/persistence/Akka.Persistence.SqlServer.Tests/SqlServerSnapshotStoreSpec.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Akka.Configuration;
+using Akka.Persistence.TestKit.Snapshot;
+
+namespace Akka.Persistence.SqlServer.Tests
+{
+    public class SqlServerSnapshotStoreSpec : SnapshotStoreSpec
+    {
+        private static readonly Config SpecConfig = ConfigurationFactory.ParseString(@"
+        akka.persistence {
+            publish-plugin-commands = on
+            snapshot-store {
+                plugin = ""akka.persistence.snapshot-store.sql-server""
+                sql-server {
+                    class = ""Akka.Persistence.SqlServer.Snapshot.SqlServerSnapshotStore, Akka.Persistence.SqlServer""
+                    plugin-dispatcher = ""akka.actor.default-dispatcher""
+                    table-name = SnapshotStore
+                    schema-name = dbo
+                    auto-initialize = on
+                    connection-string = ""Data Source=.\\SQLEXPRESS;Initial Catalog=AkkaPersistenceSqlServerSpecDb;Integrated Security=True""
```

Comment:
This needs to be configured somehow. we can't expect the build serer and users to have this specific setup.